build/lib/watch/yarn.lock

```diff
@@ -5,134 +5,162 @@
 abbrev@1:
   version "1.1.1"
   resolved "https://registry.yarnpkg.com/abbrev/-/abbrev-1.1.1.tgz#f8f2c887ad10bf67f634f005b6987fed3179aac8"
+  integrity sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==
```

Comment:
Did you run yarn with additional parameters? I am not sure whether we need this or not. @MattIrv  any idea?

Reply:
He's on a newer version of Yarn than some of us since he just installed the latest. Unless there's a reason to stay on the older version we should all probably upgrade

Reply:
👍 as I now also see this on latest yarn. Once we need to add new components, this'll be something we all have to do so might as well fix now.